uuid_shortener/shortener.py

```diff
@@ -56,3 +56,12 @@ def unshorten(self, short_uuid: str) -> UUID:
             uuid_number = left_pad + uuid_number
 
         return UUID(uuid_number)
+
+
+class ShortUuidGenerator(object):
+    def __init__(self, prefix=None, uuid=uuid4):
```

Comment:
Can you rename the the `uuid` to something that reflects the fact that it should be a function/callable, eg `uuid_fn` would be better.
Also please add a type hint to it/